index.html

```diff
@@ -44,7 +44,149 @@ <h5> I can help you build a product ,
       </div>
       <img src="images/Group.png" class="img-fluid ${3|rounded-top,rounded-right,rounded-bottom,rounded-left,rounded-circle,|}" alt="">
   </section> 
-   
+  <section class="works">
+    <h1 class="title container">My Recent Works</h1>
+    <hr>
+     <div class="image container"></div>
+     <h2 class="corner-title container">Multi-Post Stories</h2>
+     <p class="content container">A daily selection of privately personalized
+        reads; no accounts or sign-ups required.
+       has been the industry's standard dummy
+       text ever since the 1500s, when an 
+       unknown printer took a standard dummy text.</p>
+     <div class="lessons container"> 
+     <span class="html">html</span>
+     <span class="css">css</span>
+     <span class="bootstrap boot1">bootstrap</span>
+     <span class="ruby ">ruby</span>
+     </div> 
```

Comment:
- [ ] According to the project requirement, you should use the colors defined for the template.
- [ ] please change the color of each `span`. use color: #091E42 and I would suggest you make spacing after `text`. There is the spacing between paragraphs and buttons.
### your design
![his](https://user-images.githubusercontent.com/81727988/193011071-f2c65e20-c3c2-4697-bf8f-4b6ee01cafa7.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/193011122-579ad89a-f7e1-4fb5-9852-1e3a2b503816.png)